.cirrus.yml

```diff
@@ -242,53 +242,11 @@ task:
   build_script: su postgres -c "make -s -j${BUILD_JOBS} world-bin"
   upload_caches: ccache
 
-      configure_script: |
-        su postgres <<-EOF
-          ./configure \
-            --enable-cassert --enable-debug --enable-tap-tests \
-            --enable-nls \
-            \
-            ${LINUX_CONFIGURE_FEATURES} \
-            \
-            CC="ccache gcc" CFLAGS="-O0 -ggdb" \
-            CXX="ccache g++" CXXFLAGS="-O0 -ggdb" \
-            CLANG="ccache clang"
-        EOF
-      build_script: su postgres -c 'make -s -j${BUILD_JOBS} world-bin'
-      upload_caches: ccache
-
-      test_world_script: |
-        su postgres <<-EOF
-          ulimit -c unlimited # default is 0
-          make -s ${CHECK} ${CHECKFLAGS} -j${TEST_JOBS}
-        EOF
-
-    - name: Linux - Debian Bullseye - Meson
-
-      configure_script:
-        - su postgres -c 'meson setup --buildtype debug -Dcassert=true -Dssl=openssl -Duuid=e2fs build'
-      build_script:
-        - su postgres -c 'ninja -C build'
-      upload_caches:
-        - ccache
-
-      tests_script:
-        - su postgres -c 'meson test --no-rebuild -C build'
-
-    - name: Linux - Debian Sid - Meson
-
-      compute_engine_instance:
-        image: family/pg-ci-sid
-
-      configure_script:
-        - su postgres -c 'meson setup --buildtype debug -Dcassert=true -Dssl=openssl -Duuid=ossp build'
-      build_script:
-        - su postgres -c 'ninja -C build'
-      upload_caches:
-        - ccache
-
-      tests_script:
-        - su postgres -c 'meson test --no-rebuild -C build'
```

Comment:
Re changes shouldn't be here, in a prior comment: Note that you're removing the changes you made in that prior commit here.